cstrml/tee/src/lib.rs

```diff
@@ -370,13 +370,38 @@ impl<T: Trait> Module<T> {
 
         Used::put(total_used);
         Reserved::put(total_reserved);
+        let total_workload = total_used + total_reserved;
 
-        // 5. Call `on_report_works` handler
-        T::Works::report_works(
-            &who,
-            used + reserved,
-            total_used + total_reserved,
-        );
+        // 5. Update work report for every identity
+        let workload_map: Vec<(T::AccountId, u128)> = <TeeIdentities<T>>::iter().map(|(controller, _)| {
```

Comment:
Do not need iterate all identities, cause this only influent these contains work report